pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -0,0 +1,1018 @@
+{
+  "aci": {
+    "owner": "terraform-providers",
+    "repo": "terraform-provider-aci",
+    "rev": "v0.2.3",
+    "sha256": "0sk0pp178w03fhsb65b9mpim1l4wqfnv9r9x64kiapjnvfb1rz3j",
+    "version": "0.2.3"
+  },
+  "acme": {
+    "owner": "getstackhead",
+    "provider-source-address": "registry.terraform.io/getstackhead/acme",
+    "repo": "terraform-provider-acme",
+    "rev": "v1.5.0-patched",
+    "sha256": "1wdrjpd3l0xadsa3lqhsc9c57g8x2qkwb76q824sk8za1a7lapii",
+    "version": "1.5.0-patched"
+  },
```

Comment:
I have been using the `acme` provider to test. The remaining `"provider-source-address"` will be added within this PR.